django_components/component.py

```diff
@@ -84,6 +86,11 @@ def __init__(
         self.outer_context: Context = outer_context or Context()
         self.fill_content = fill_content
 
+    @classmethod
+    @property
+    def class_hash(cls):
+        return hash(str(inspect.getfile(cls)) + str(cls.__name__))
+
```

Comment:
Using inspect here bugs me a bit. I think it's because I've mostly used inspect when debugging or printing exceptions. I wonder if we could simply use `cls.__module__ + cls.__name__` here instead?

Reply:
Well researched, and explained, thanks. Let's go with your solution!